### ISSUE TITLE
Issue GridapGmsh in Windows 10

### ISSUE THREAD

**Issue by c-schubert** [gridap/GridapGmsh.jl#36]
Query of deps/build.jl line 20 and 21 does not work under Windows 10, even if the environment variable to the GMSH SDK is set correctly.
``` Julia
 gmsh_bin = joinpath(gmsh_root,"bin","gmsh")
  if isfile(gmsh_bin)
```
Fix is:
``` Julia
 gmsh_bin = joinpath(gmsh_root,"bin","gmsh.exe")
  if isfile(gmsh_bin)
```
Of course, a query of the target system should be made before ,so that at the same the functionality for Unix systems is not disturbed ...

**Comment by fverdugo** [comment]
Thanks for reporting! Perhaps we can check that either gmsh or gmesh.exe exists. Can you open a PR?

**Comment by c-schubert** [comment]
Done but found another issue, I do not know how to fix in a hurry ... , will open another issue in this ...

**Comment by fverdugo** [comment]
Fixed in PR #37
